elliott/elliottlib/bzutil.py

```diff
@@ -290,7 +311,9 @@ def version(self):
 
     @property
     def blocked_by_bz(self):
-        url = getattr(self.bug.fields, JIRABugTracker.field_blocked_by_bz)
+        if not self.jira_client:
+            self.jira_client = get_jira_client()
+        url = getattr(self.bug.fields, get_jira_field_id(self.jira_client, "Blocked by Bugzilla Bug"))
```

Comment:
I don't particularly like this pattern. We should fail if there is no jira client stored in bug
And in all methods of JiraBugTracker where we are fetching bugs, we store jira client in every jira bug